app/controllers/compute_resources_controller.rb

```diff
@@ -36,6 +36,11 @@ def create
       @compute_resource.valid?
       process_error
     end
+  rescue StandardError => e
+    logger.error "Error while creating ovirt resource: #{e.message}\n#{e.backtrace.join("\n")}"
```

Comment:
This isn't necessarily an ovirt resource

Reply:
Right. when an exception comes back to the controller, I do not want it to return an exception screen, or 500 page, but to display an error message to the user.

If an exception was not handled prior to that location, at least allow the user to continue doing other stuff.

Reply:
@tbrisker was probably referring to the `ovirt` word in `Error while creating ovirt resource'`, it should be `Error while creating compute resource`

Reply:
waiting for the PR of mine about unified Ovirt exception, and then I'll change this change accordingly.

Reply:
I agree with @tbrisker.
Lets keep the error message general and in the next PR where ovirt exception are thrown, add a more explicit error message saying it was `ovirt`.

WDYT?

Reply:
I was indeed referring to the error message here. This is the compute resource controller which is used by all compute resources. Any error message should not contain a reference to ovirt, since that won't make sense, for example, when someone hits an error when trying to create a vmware resource.

Reply:
sorry for that, you are right, and now added support for it.